flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/test/program/TableTestProgram.java

```diff
@@ -198,6 +198,18 @@ public SqlTestStep getRunSqlTestStep() {
         return (SqlTestStep) sqlSteps.get(0);
     }
 
+    /**
+     * Convenience method to avoid boilerplate code. It assumes only one statement set is tested.
+     */
+    public StatementSetTestStep getRunStatementSetTestStep() {
+        List<TestStep> statementSetSteps =
+                runSteps.stream()
+                        .filter(s -> s.getKind() == TestKind.STATEMENT_SET)
+                        .collect(Collectors.toList());
+
+        return (StatementSetTestStep) statementSetSteps.get(0);
```

Comment:
Any particular reason to assume that there is only one StatementSet being tested?

Reply:
For tests, as a convenience we can assume there will be only 1 statement set which can contain multiple sql statements.

Reply:
If we make that assumption, can we verify it similarly as we do for `getRunSqlTestStep`

Reply:
Added a precondition check to verify similar to `getRunSqlTestStep`